image-loader/app/model/ImageUpload.scala

```diff
@@ -119,4 +160,13 @@ case object ImageUpload {
     )
   }
 
+  private def isPng24(mimeType: Option[String], fileMetadata: FileMetadata): Boolean =
+    mimeType match {
+      case Some("image/png") => {
+        val colourType = fileMetadata.colourModelInformation.get("colorType").getOrElse("")
```

Comment:
can just be `val colourType = fileMetadata.colourModelInformation.getOrElse("colorType", "")`